src/python_testing/TC_RVCCLEANM_2_1.py

```diff
@@ -84,19 +84,40 @@ async def test_TC_RVCCLEANM_2_1(self):
         self.endpoint = self.matter_test_config.endpoint
         self.mode_ok = self.matter_test_config.global_test_params['PIXIT.RVCCLEANM.MODE_CHANGE_OK']
         self.mode_fail = self.matter_test_config.global_test_params['PIXIT.RVCCLEANM.MODE_CHANGE_FAIL']
-        self.is_ci = self.check_pics("PICS_SDK_CI_ONLY")
-        if self.is_ci:
-            app_pid = self.matter_test_config.app_pid
-            if app_pid == 0:
-                asserts.fail("The --app-pid flag must be set when PICS_SDK_CI_ONLY is set")
+        app_pid = self.matter_test_config.app_pid
+        if app_pid != 0:
```

Comment:
I know this isn't your code, but I'm also not sure this is correct. If the flag isn't set, won't app_pid be None?

Reply:
Hi Cecille,
So when I run this in the Raspberry pi Dev container it was returning a 0 for app_pid if --app-pid is not provided in the CLI arguments. 
In the matter_testing_support module it appears to set the possible variables to 0 if they are to be gathered by argparse and not found in the provided command line arguments:
- config.app_pid = 0 if args.app_pid is None else args.app_pid